src/hotspot/share/opto/loopnode.cpp

```diff
@@ -1859,6 +1849,38 @@ bool PhaseIdealLoop::is_counted_loop(Node* x, IdealLoopTree*&loop, BasicType iv_
     }
   }
 
+  Node *sfpt2 = NULL;
```

Comment:
```suggestion
  Node* sfpt2 = NULL;
```

Reply:
I would suggest to rename `sfpt2` to `sfpt` and `sfpt` below to `backedge_sfpt`.